contents/handbook/people/spending-money.md

```diff
@@ -153,13 +156,28 @@ We have a company WeWork All Access account that allows you to book desks in any
 
 ## Travel
 
-If you need to travel on PostHog's behalf (e.g. in person onboarding, meeting customers, working on a specific project), again please spend money in the best interests of the company. We do not pre-approve your travel expenses, and trust team members to make the right decisions here. Some guidance:
-
-- We all travel economy by default - we cannot afford for folks to fly premium or business class. Feel free to upgrade using your personal money/airmiles if you'd like to. If you are particularly tall and struggle to fit into an economy seat, PostHog will cover the cost of an emergency exit seat. 
-- Be sensible when considering the value of your time, and avoid _over-optimizing_ for cost. It is probably not worth saving $100 to take a non-direct flight that takes you 5hrs extra. Similarly, it doesn't make sense to save $50 by taking a flight that leaves at 6am, only to have to expense a taxi to get to the airport. 
-- Related to the above, consider signing up for programs like Global Entry if you are regularly travelling to countries that offer it - these can pay for themselves in one trip with the amount of time saved, particularly when travelling to the US. PostHog will cover costs here. 
-
-If you have any questions about this stuff, or are unsure what is reasonable here, feel free to check in with Charles. 
+If you need to travel on PostHog’s behalf (e.g. in person onboarding, meeting customers, working on a specific project), again please spend money in the best interests of the company. 
+
+We now use Brex Travel to help with planning and booking travel. If you are booking an individual trip, you can use the ‘Travel’ function to search and book flights directly through Brex.
+- First things first, click the trips icon and go to ‘Manage Profile’. Here you can enter all of your travel details, including loyalty program and known traveler numbers, so you never have to enter them manually with an airline, hotel, or car hire service ever again
```

Comment:
@KendalHall can you edit this section for tone so it's a bit more formal and less tongue-in-cheek?